misskey.tera

```diff
@@ -0,0 +1,12 @@
+---
+whiskers:
+  version: "2.3.0"
+  matrix:
+   - accent
+   - flavor
+  filename: "src/themes/{{ flavor.identifier }}/catppuccin-{{ flavor.identifier }}-{{ accent }}.json5"
```

Comment:
```suggestion
  filename: "themes/{{ flavor.identifier }}/catppuccin-{{ flavor.identifier }}-{{ accent }}.json5"
```
I'd like to change it to this after this PR, keeping it the same for a minimal diff for now.